### DISCUSSION TITLE
企业开源该选什么软件许可证？ | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#55] [Comments]
# 企业开源该选什么软件许可证？ | 夜天之书

开源软件和自由软件的概念与其许可证紧密绑定。 通常，开源软件被定义为使用 OSI 认可的，即符合开源定义的许可证来分发的软件，而自由软件被定义成使用 GPL 或说 Copyleft 式许可证分发的软件。 尽管今天人们最关心的可能是软件的生产过程即如何进行开源协同，但是为软件选择什么样的许可证，仍然是一个项目启动时必须考虑和决定的问题。某种意义上说，软件可以被如何使用和分发，反过来约束了可能实现的协

https://www.tisonkun.org/2022/12/17/enterprise-choose-a-software-license/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

许可协议渐纷繁
权益保护难分明
自由开源非常态
良善心意得保护
